tests/test_basic.py

```diff
@@ -160,8 +160,8 @@ def test_dtstamp(tmpdir, runner, create):
     result = runner.invoke(cli, ['new', '-l', 'default', 'test event'])
     assert not result.exception
 
-    db = Database(str(tmpdir + '/default'))
-    todo = list(db.todos.values())[0]
+    db = Database([str(tmpdir + '/default')], str(tmpdir + '/dtstamp_cache'))
```

Comment:
It's more readable, and I don't mind the refactor now, since it doesn't add noise to the project log anyway (it gets squashed).